### DISCUSSION TITLE
github.com/settings/profile/location -> Time Zone, Incorrect time range for Dublin

### DISCUSSION THREAD

**Discussion by c4rt0** [community/community#61368] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

In the profile page when setting up Time Zone for Dublin user is simply presented with: `(GMT+01:00) Dublin` throughout the year.

AFAIK this is incorrect since Ireland uses Irish Standard Time (IST, GMT+01:00) in the summer months and Greenwich Mean Time (GMT+00:00) in the winter period.

Simple logic should be implemented to present the user with the correct GMT for the appropriate period.

**Comment by RamtinKosari** [comment, marked as answer]
Hi Dear @c4rt0 
Dublin, Ireland operates on Irish Standard Time (IST) during the non-daylight saving time period. The time zone for Dublin during daylight saving time is Irish Summer Time (IST), which is one hour ahead of Irish Standard Time.

You are correct. Ireland follows Irish Standard Time (IST) during the summer months, which is GMT+01:00, and Greenwich Mean Time (GMT) during the winter period, which is GMT+00:00. Therefore, it is not accurate to say that Dublin remains on GMT+01:00 throughout the year.

To accurately represent the time zone for Dublin, the profile page should include both time zones: (GMT+01:00) Dublin (Irish Standard Time - IST) in the summer and (GMT+00:00) Dublin (Greenwich Mean Time - GMT) in the winter. This would reflect the time changes observed in Ireland throughout the year.

i've created an issue on related repo of github, hope that they will fix this bug

**Reply by BBDBABDBU** [reply]
I can fix this problem but i dont know where i should contribute to their project
